web/themes/custom/itkdev/itkdev_project_theme/templates/components/new-project-button.html.twig

```diff
@@ -0,0 +1,4 @@
+<a href="/node/add/project" class="font-thin bg-green-700 hover:bg-green-600 text-white py-2 px-4 rounded inline-flex items-center" data-drupal-link-system-path="node/add/project">
```

Comment:
Consider using `path` to get the `href` value (like you usually do in Twig):

```suggestion
{% set path = path('node.add', {node_type: 'project'}) %}
<a href="{{ path }}" class="font-thin bg-green-700 hover:bg-green-600 text-white py-2 px-4 rounded inline-flex items-center" data-drupal-link-system-path="{{ path }}">
```

(`drush route | grep node/add` reveales the route name to use)